src/sentry/static/sentry/app/components/avatarCropper.jsx

```diff
@@ -37,7 +39,7 @@ class AvatarCropper extends React.Component {
     /*eslint consistent-return:0*/
     let file = ev.target.files[0];
 
-    if (!file) return; // No file selected (e.g. user clicked "cancel")
+    if (!file) return; // No file selected (e.g. model clicked "cancel")
```

Comment:
i think it's still the user clicking cancel (assume this was just replace all `user` with `model`)